string freq_m1
string freq_m2
string freq_m3
string freq_m4
string distancia